mlir/include/mlir/Dialect/SparseTensor/Transforms/Passes.td

```diff
@@ -123,6 +123,18 @@ def SparsificationPass : Pass<"sparsification", "ModuleOp"> {
   ];
 }
 
+def StageSparseOperations : Pass<"stage-sparse-ops", "func::FuncOp"> {
```

Comment:
This should be in the same order as header file, so either here before  PostSparsificationRewriting or in header file at end

Reply:
Never mind. I am still getting used to the new diff tools and this looked like it was at EOF.
So, good to go!